src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -63,7 +66,12 @@ public async Task OnStartProcess(string startEvent, Instance instance)
         /// <inheritdoc />
         public async Task OnEndProcess(string taskId, Instance instance)
         {
-            _logger.LogInformation($"OnEndProcess for {instance.Id}");
+            _logger.LogInformation($"OnEndProcess for {instance.Id}, taskId: {taskId}");
+
+            if (taskId != null && taskId.Equals("EndEvent_1"))
```

Comment:
will taskId ever be endEvent?? Wouldn't that just be an event?

Reply:
also., if not we might need to update the default appmetadata created by designer? it has now set that pdf is to be added during task_1 if I remember correctly

Reply:
Yes, EndEvent_1 is what is passed in now.

Ah, good catch. I'll update the template 👍

Reply:
But I'm unsure if it really matters if the Task_1 is added in the appmetadata, this code works in at21 now.

Is this validated in any way? 

Either way, the template should be with endevent.

Reply:
hmm, strange. It should have caught the error before the pdf is posted to storage..